parts/k8s/kubernetescustomscript.sh

```diff
@@ -167,10 +162,6 @@ if $FULL_INSTALL_REQUIRED; then
         # mitigation for bug https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1676635
         echo 2dd1ce17-079e-403c-b352-a1921ee207ee > /sys/bus/vmbus/drivers/hv_util/unbind
         sed -i "13i\echo 2dd1ce17-079e-403c-b352-a1921ee207ee > /sys/bus/vmbus/drivers/hv_util/unbind\n" /etc/rc.local
```

Comment:
this was also moved in the FULL_INSTALL block in https://github.com/Azure/acs-engine/pull/3742/files.... Should we move it out completely?

Reply:
Out of scope.